spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -21,5 +22,57 @@ module RSpec::Rails
 
       expect { group.new.setup_fixtures }.to_not raise_error
     end
+
+    context "without database available" do
+      let(:example_group) do
+        RSpec::Core::ExampleGroup.describe("FixtureSupport") do
+          include FixtureSupport
+          include RSpec::Rails::MinitestLifecycleAdapter
+        end
+      end
+      let(:example) do
+        example_group.example("foo") do
+          expect(true).to be(true)
+        end
+      end
+
+      RSpec.shared_examples_for "unrelated example raise" do
+        it "raise due to no connection established" do
+          expect(example_group.run).to be(false)
+          expect(example.execution_result.exception).to \
+            be_a(ActiveRecord::ConnectionNotEstablished)
+        end
+      end
+
+      RSpec.shared_examples_for "unrelated example does not raise" do
+        it "does not raise" do
+          expect(example_group.run).to be(true)
+          expect(example.execution_result.exception).not_to \
+            be_a(ActiveRecord::ConnectionNotEstablished)
+        end
+      end
+
+      before { clear_active_record_connection }
+
+      after { establish_active_record_connection }
+
+      context "with use_active_record set to false" do
+        before { RSpec.configuration.use_active_record = false }
+
+        after { RSpec.configuration.use_active_record = true }
+
+        include_examples "unrelated example does not raise"
```

Comment:
This spec does nothing to demonstrate that the support works when active record is available but not configured, you should be able to demonstrate the issue without adding database support as its supposed to be broken without database support.

Reply:
Unfortunately, I don't get your point. When `use_active_record` is set to `true` and no AR connection is available, it raises an `ActiveRecord::ConnectionNotEstablished`. When `use_active_record` is set to `false` and no AR connection is available it does not raise. Could you explain in detail what you have in mind for another test?

Reply:
Its artificial because you have configured a database then manually cleared it.

The default is no configuration, the changes adding config should be removed and these spec should demonstrate that `requiring' active_record'` causes an error, and that setting `use_active_record` false prevents it.

Reply:
Sorry, I've been trying to find time to work on this myself because I believe strongly that adding database configuration for all the specs is not the way to go! :)